scripts/start.sh

```diff
@@ -112,7 +112,7 @@ echo Starting GAE development server
 # settings in feconf.py. Be careful with this -- you do not want to spam people
 # accidentally!
 
-if [[ "$MINIFICATION" == "true" ]]; then
+if [[ "$DEV_MODE" == "false" ]]; then
```

Comment:
Shouldn't this be `"$FORCE_DEV_MODE" == "False"`?

Reply:
got it changed to `"$FORCE_PROD_MODE" =="True"`.